sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/factory/AbstractAzureServiceClientBuilderFactory.java

```diff
@@ -36,6 +38,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractAzureServiceClientBuilderFactory.class);
     private static final TokenCredential DEFAULT_TOKEN_CREDENTIAL = new DefaultAzureCredentialBuilder().build();
+    private static final AzureTokenCredentialResolver DEFAULT_TOKEN_CREDENTIAL_RESOLVER = new AzureTokenCredentialResolver();
```

Comment:
Wouldn't it be better if both used the bean `AzureTokenCredentialResolver` ? Because the default `AzureTokenCredentialResolver` does not apply the functionality of the AzureAadCredentialBuilderFactory.